docsrc/conf.py

```diff
@@ -14,19 +14,22 @@
 #
 import os
 import sys
+
 sys.path.insert(0, os.path.abspath('.'))
 sys.path.append(os.path.join(os.path.dirname(__name__), '..'))
 
+with open(os.path.join('..', 'mabwiser', '_version.py')) as fp:
```

Comment:
Good idea to also use this logic in `conf.py`! I might do the same the next time I update TextWiser documentation.